community-operators/eclipse-che/7.24.0/eclipse-che.v7.24.0.clusterserviceversion.yaml

```diff
@@ -0,0 +1,1206 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  annotations:
+    alm-examples: |-
+      [
+        {
+          "apiVersion": "org.eclipse.che/v1",
+          "kind": "CheCluster",
+          "metadata": {
+            "name": "eclipse-che"
+          },
+          "spec": {
+            "auth": {
+              "externalIdentityProvider": false,
+              "identityProviderAdminUserName": "",
+              "identityProviderClientId": "",
+              "identityProviderImage": "",
+              "identityProviderPassword": "",
+              "identityProviderRealm": "",
+              "identityProviderURL": "",
+              "oAuthClientName": "",
+              "oAuthSecret": ""
+            },
+            "database": {
+              "chePostgresDb": "",
+              "chePostgresHostName": "",
+              "chePostgresPassword": "",
+              "chePostgresPort": "",
+              "chePostgresUser": "",
+              "externalDb": false,
+              "postgresImage": ""
+            },
+            "metrics": {
+              "enable": true
+            },
+            "server": {
+              "allowUserDefinedWorkspaceNamespaces": false,
+              "cheClusterRoles": "",
+              "cheFlavor": "",
+              "cheImage": "",
+              "cheImageTag": "",
+              "cheWorkspaceClusterRole": "",
+              "devfileRegistryImage": "",
+              "gitSelfSignedCert": false,
+              "nonProxyHosts": "",
+              "pluginRegistryImage": "",
+              "proxyPassword": "",
+              "proxyPort": "",
+              "proxyURL": "",
+              "proxyUser": "",
+              "serverExposureStrategy": "",
+              "serverMemoryLimit": "",
+              "serverMemoryRequest": "",
+              "serverTrustStoreConfigMapName": "",
+              "singleHostGatewayConfigMapLabels": {},
+              "singleHostGatewayConfigSidecarImage": "",
+              "singleHostGatewayImage": "",
+              "tlsSupport": true,
+              "useInternalClusterSVCNames": true,
+              "workspaceNamespaceDefault": ""
+            },
+            "storage": {
+              "postgresPVCStorageClassName": "",
+              "preCreateSubPaths": true,
+              "pvcClaimSize": "1Gi",
+              "pvcJobsImage": "",
+              "pvcStrategy": "common",
+              "workspacePVCStorageClassName": ""
+            }
+          }
+        }
+      ]
+    capabilities: Seamless Upgrades
+    categories: Developer Tools, OpenShift Optional
+    certified: "false"
+    containerImage: quay.io/eclipse/che-operator@sha256:c4fbaf0a44ae88afe7fd9dabdb65919315fd8f9b78a6a2a7914e62e5b3657ac5
```

Comment:
unable to pull the image

Reply:
working for me here... also as `quay.io/eclipse/che-operator:7.24.0` 

Try your test again?